app.js

```diff
@@ -175,18 +175,20 @@ function doCommand(command, client, message) {
             }
 
         } else {
-            console.warn("Command: " + command.name + " | had an error. Show the developer of the command module that you are getting this error code: \n" + err)
+            console.warn("Command: " + command.name + " | had an error. Show the client.developer of the command module that you are getting this error code: \n" + err)
```

Comment:
woops, need to change this to "developer" instead of "client.developer"

[selfreminder]

Reply:
Fixed in latest commit.